src/components/dashboard/inventory/InventoryEdit.js

```diff
@@ -0,0 +1,119 @@
+                                    delimiters={[32, 44, 13, 9]}
+                                    classNames={{
+                                        tags: 'tagsClass',
+                                        tagInput: 'w-100 ma1',
+                                        tagInputField: 'w-100 pa1',
+                                        selected: 'selectedClass',
+                                        tag: 'ml1 pa1 mv2 f6 b--black-20 ba',
+                                        remove: 'b pa1',
+                                        suggestions: 'suggestionsClass'
+                                    }}
+                                    />
+                            </div>
+                            <div className="w-50-l w-100 dt f6 pa2 fr">
+                                <div className="w-50 dtc v-mid pa1">
+                                    <label className={labelClass}>Bean Image:</label>
+                                    <div className="ma1 ph5  ba">
+                                        <BeanItemImage src={this.props.image || item.pictureURL}/>
+                                    </div>
+                                </div>
+                                <div className="w-50 dtc v-mid pa1">
+                                    <FileSelector
+                                        fileSelected={this.props.photo}
+                                        buttonText={'Upload Image'}
+                                        />
+                                </div>
+                            </div>
+                            <div className="w-50-l w-100 pa2 dib">
+                                <label className={labelClass}>Description:</label>
+                                <textarea id="decaf" ref={this.props.description} className={inputClass} value={item.description} required/>
+                            </div>
+                            <div className="pa3 pa3-ns w-100">
+                                <input className="self-center b ph3 pv2 input-reset ba b--black bg-white grow pointer f6 dib" type="submit" value="Add Roast"/>
+                            </div>
+                        </div>
+                    </fieldset>
+                </form>
+            </div>
+        );
+    }
+}
+
+InventoryEdit.propTypes = {
+    fetching: PropTypes.bool.isRequired,
+    success: PropTypes.bool.isRequired,
+    onAddBeans: PropTypes.func.isRequired,
+    name: PropTypes.func.isRequired,
+    type: PropTypes.func.isRequired,
+    bags: PropTypes.func.isRequired,
+    size: PropTypes.func.isRequired,
+    price: PropTypes.func.isRequired,
+    isDecaf: PropTypes.func.isRequired,
+    isActive: PropTypes.func.isRequired,
+    description: PropTypes.func.isRequired,
+    tags: PropTypes.array.isRequired,
+    photo: PropTypes.func.isRequired,
+    image: PropTypes.string,
+    onDeleteTag: PropTypes.func.isRequired,
+    onAddTag: PropTypes.func.isRequired
+};
```

Comment:
If we do this, we could have something similar to how we're handling registration/updating account information. If the row object is null, have the default values for inputs be an empty string, otherwise have it be whatever it is in the row.